Python/pywarpx/picmi.py

```diff
@@ -450,9 +450,9 @@ def initialize_inputs(self, species_number, layout, species, density_scale):
         species.multiple_particles_pos_x = self.x
         species.multiple_particles_pos_y = self.y
         species.multiple_particles_pos_z = self.z
-        species.multiple_particles_vel_x = np.array(self.ux)/constants.c
-        species.multiple_particles_vel_y = np.array(self.uy)/constants.c
-        species.multiple_particles_vel_z = np.array(self.uz)/constants.c
+        species.multiplt_particle_ux = np.array(self.ux)/constants.c
```

Comment:
I notice a typo here - this should be "multiple_particle_ux", not "multiplt_particle_ux". I see the same typo every place this substitution is made.

Reply:
Thanks for checking 😓